tests/test_openai_chat_director.py

```diff
@@ -289,6 +289,137 @@ def test_answer_not_known_using_notes_command(chat_client_no_background, default
     assert response_message == prompts.no_entries_found.format()
 
 
+# ----------------------------------------------------------------------------------------------------
+@pytest.mark.django_db(transaction=True)
+@pytest.mark.chatquality
+def test_summarize_one_file(chat_client, default_user2: KhojUser):
+    message_list = []
+    conversation = create_conversation(message_list, default_user2)
+    # post "Xi Li.markdown" file to the file filters
+    file_list = (
+        Entry.objects.filter(user=default_user2, file_source="computer")
+        .distinct("file_path")
+        .values_list("file_path", flat=True)
+    )
+    # pick the file that has "Xi Li.markdown" in the name
+    summarization_file = ""
+    for file in file_list:
+        if "Birthday Gift for Xiu turning 4.markdown" in file:
+            summarization_file = file
+            break
+    assert summarization_file != ""
+
```

Comment:
So this portion of the code is to get the proper path because this will be slightly different for a user testing locally. There might be a better way to do this but this was the simplest way for me to get the path containing the appropriate file.